### PR TITLE
feat: added sign up and login page.

### PR DESCRIPTION
**Pull request by xii-vi**
### login
Login-RE-KICKS

### sign-up
Register-RE-KICKS

### CODE REVIEW

File: stylesheet/login.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    width: 100%;
+}
+
+form{
+    background-color: whitesmoke;
+    height: 35rem;
+    margin: 2rem;
+    background-image: url(/assests/6111326.webp);
+    background-size: cover;
+}
```
Comment (bishnoimukesh) [inline stylesheet/login.css]: avoid using html tags directly

File: stylesheet/signUP.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    background-image: url(/assests/24c452cf0422adb434a058e87fced33b.jpg);
+    background-repeat: repeat;
+}
+form{
+    background-image: none;
+    background-color: var(--primary-color);
+    color: var(--white-color);
+    width: 40rem;
+    margin: 0;
+}
```
Comment (bishnoimukesh) [inline stylesheet/signUP.css]: same

**Review by bishnoimukesh — COMMENTED** [review]
change suggested rest looks good

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Review by haarsh24 — APPROVED** [review]
Good to merge .

**Pull request merged**